src/api/venue.ts

```diff
@@ -77,3 +79,65 @@ export const updateIframeUrl = async (iframeUrl: string, venueId?: string) => {
     .functions()
     .httpsCallable("venue-adminUpdateIframeUrl")({ venueId, iframeUrl });
 };
+
+const getUserInSectionRef = (
+  userId: string,
+  venueId: string,
+  sectionId: string
+) =>
+  firebase
+    .firestore()
+    .collection("venues")
+    .doc(venueId)
+    .collection("sections")
+    .doc(sectionId)
+    .collection("seatedUsers")
+    .doc(userId);
+
+export const unsetAuditoriumSectionSeat = async (
+  userId: string,
+  venueId: string,
+  sectionId: string
+) => {
+  return getUserInSectionRef(userId, venueId, sectionId)
+    .delete()
+    .catch((err) => {
+      Bugsnag.notify(err, (event) => {
+        event.addMetadata("context", {
+          location: "api/venue::unsetAuditoriumSectionSeat",
+          venueId,
+          sectionId,
+          userId,
+        });
+      });
+
+      throw err;
+    });
+};
+
+export const setAuditoriumSectionSeat = async (
+  user: WithId<DisplayUser>,
+  venueId: string,
+  sectionId: string,
+  row: number,
+  column: number
+) => {
+  return getUserInSectionRef(user.id, venueId, sectionId)
+    .set({
+      ...pickDisplayUserFromUser(user),
```

Comment:
why do you want to spread its data? Why not just store it as `user`

Reply:
For `WithId` stuff in selectors to work properly
I've tried to make a separate field `user` - it complicates things.